web/webkit/src/main/scala/net/liftweb/http/LiftSession.scala

```diff
@@ -2325,32 +2326,68 @@ class LiftSession(private[http] val _contextPath: String, val underlyingId: Stri
    * the CometActor is instantiated.  If the CometActor already exists
    * in the session, the message will be sent immediately.  If the CometActor
    * is not yet instantiated, the message will be sent to the CometActor
-   * as part of setup (@see setupComet) if it is created as part
+   * as part of setup (@see queueCometMessage) if it is created as part
```

Comment:
Inline, `@see` doesn't actually work. This could be `(see `[[queueCometMessage]]`)` perhaps? With backticks before and after the square brackets, but I can't do that in inline markdown-land seemingly.